feature/about/src/main/java/io/github/droidkaigi/confsched2020/about/ui/AboutFragment.kt

```diff
@@ -87,7 +89,13 @@ class AboutFragment : Fragment(R.layout.fragment_about), Injectable {
                 aboutItemFactory.create(
                     getString(R.string.about_item_access)
                 ) {
-                    // TODO go access-page
+                    val intent = Intent(
```

Comment:
Currently, we use `SystemViewModel` for launching intent. 🙏

Reply:
Fixed.